README.md

```diff
@@ -81,21 +130,28 @@ To add RAC to your application:
  1. If your application target does not contain Swift code at all, you should also
     set the `EMBEDDED_CONTENT_CONTAINS_SWIFT` build setting to “Yes”.
 
-Or, if you’re using [Carthage](https://github.com/Carthage/Carthage), simply add
-ReactiveCocoa to your `Cartfile`:
 
-```
-github "ReactiveCocoa/ReactiveCocoa"
-```
-Make sure to add `ReactiveCocoa.framework`, `ReactiveSwift`, and `Result.framework` to "Linked Frameworks and Libraries" and "copy-frameworks" Build Phases.
+## Objective-C, Swift and the ReactiveCocoa family
+
+Starting with [version 5.0][CHANGELOG], ReactiveCocoa primarily focuses on providing reactive extensions to Cocoa frameworks in Swift. In other words:
+
+1. The core Swift API - which provides FRP inspired primitives - had been spun off as [ReactiveSwift][].
+
+2. The legacy Objective-C API had been spun off as [ReactiveObjC][].
+
+3. The bridge between the core Swift API and the legacy Objective-C API is now hosted in ([ReactiveObjCBridge][]), which is mostly meant as a compatibility layer for older ReactiveCocoa projects.
+
+The Objective-C API will continue to be maintained and supported in the foreseeable future. But given the maturity of the framework and the language, non-trivial changes are not expected. Please visit [ReactiveObjC][] for more information.
+
+**We highly recommend all Swift projects using the Swift API.**
```

Comment:
This section could also be consolidated and put up top. I'm thinking that 1-2 paragraphs explaining ReacitveSwift and ReactiveObjC should be sufficient.